code/controllers/subsystem/tickets/mentor_tickets.dm

```diff
@@ -10,19 +10,25 @@ GLOBAL_REAL(SSmentor_tickets, /datum/controller/subsystem/tickets/mentor_tickets
 	ticket_system_name = "Mentor Tickets"
 	ticket_name = "Mentor Ticket"
 	span_class = "mentorhelp"
+	anchor_link_extra = ";is_mhelp=1"
+	ticket_help_type = "Mentorhelp"
+	ticket_help_span = "mentorhelp"
 	other_ticket_name = "Admin"
 	other_ticket_permission = R_ADMIN
 	close_rights = R_MENTOR | R_ADMIN
 	rights_needed = R_MENTOR | R_ADMIN | R_MOD
 
 /datum/controller/subsystem/tickets/mentor_tickets/Initialize()
+	. = ..()
 	close_messages = list("<font color='red' size='3'><b>- [ticket_name] Closed -</b></font>",
 				"<span class='boldmessage'>Please try to be as descriptive as possible in mentor helps. Mentors do not know the full situation you're in and need more information to give you a helpful response.</span>",
 				"<span class='[span_class]'>Your [ticket_name] has now been closed.</span>")
-	return ..()
 
```

Comment:
I am unsure of the implications of moving this up to the start of the init instead of the end
```dm
/datum/controller/subsystem/Initialize(start_timeofday)
	initialized = TRUE
	var/time = (REALTIMEOFDAY - start_timeofday) / 10
	var/msg = "Initialized [name] subsystem within [time] second[time == 1 ? "" : "s"]!"
	to_chat(world, "<span class='boldannounce'>[msg]</span>")
	log_world(msg)
	return time
```

There is a chance this can break things by marking it as initialized before it should. Personally here, I would rather have these defined on the subsystem itself, and do a check in `/datum/controller/subsystem/tickets/Initialize()` to see if `close_messages` has any entries, though I will still defer to the other maints.

Reply:
Just noticed I forgot to push the changes. They're pushed now